submissions/migrations/0003_submission_status.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0002_auto_20151119_0913'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='submission',
+            name='status',
+            field=models.CharField(default=b'A', max_length=1, choices=[(b'D', b'Deleted'), (b'A', b'Active')]),
+        ),
+    ]
```

Comment:
We're going to have to make a call here on feasibility (@e0d devops and @scottrish product).

Using the search linked below, we're looking at 500-1000 requests per hour, or on the order of tens per minute. @maxrothman and I have estimated that, given that request rate and the above timing data for a database not under load, we're looking at an upper bound of several hundred request failures during the migration. Is this an acceptable outage? It will only apply to users who are hitting the submissions database, which is only used by ORA and has a cache that will remain unaffected.

http://splunk.edx.org/en-US/app/search/search?earliest=-7d%40h&latest=now&q=search%20index%3D%22prod-edx%22%20sourcetype%3Dedx%20%22submissions.api%22%20NOT%20%22(cached)%22%20%7C%20timechart%20span%3D1h%20count%20by%20log_level&display.page.search.tab=visualizations&display.general.type=visualizations&sid=1455135927.4182068